.expeditor/verify_linux_pipeline.yml

```diff
@@ -0,0 +1,56 @@
+  - label: "[@@plan@@] :linux: :habitat: Check for bad patterns"
+    command:
+      - bin/ci/check-bad-patterns.sh @@plan@@
+    expeditor:
+      executor:
+        docker:
+    timeout_in_minutes: 10
+    retry:
+      automatic:
+        limit: 1
+
+  - label: "[@@plan@@] :linux: :habitat: Shellcheck"
+    command:
+      - bin/ci/shellcheck.sh @@plan@@
+    expeditor:
+      executor:
+        docker:
+    timeout_in_minutes: 10
+    retry:
+      automatic:
+        limit: 1
+
+  - label: "[@@plan@@] :linux: :habitat: Check for default variables"
+    command:
+      - bin/ci/check-default-variables.sh @@plan@@
+    expeditor:
+      executor:
+        docker:
+    timeout_in_minutes: 10
```

Comment:
10 minutes is the default timeout enforced by Expeditor+Buildkite. You could technically leave this off.

Reply:
We could also discuss using global defaults. In your bash script, before you print `steps:`, you could print this: 

```yaml
expeditor:
  defaults:
    buildkite:
      timeout_in_minutes: 10
      retry:
        automatic:
          limit: 1
```

`expeditor.defaults.buildkite` takes the nested settings and applies them to every job you define.

Reply:
I've re-added the `verify.pipeline.yml` that contains the above, along with the `steps: ` line and now cat that files before generating the steps.